.github/workflows/deploy.yml

```diff
@@ -24,6 +24,8 @@ jobs:
             default_max_vcpus: 600
             expanded_max_vcpus: 1600
             required_surplus: 1000
+            permission_boundary_arn: ''
+            deploy_lamdas_in_vpc: false
```

Comment:
```suggestion
            permissions_boundary_arn: ''
            deploy_lambdas_in_vpc: false
```

Reply:
Both the the matrix field names are typoed. They're consistently typoed, so it'll work, but I'd prefer to see them fixed throughout.